### ISSUE TITLE
NewPipe merchandise for conferences, developer meetings etc.

### ISSUE THREAD

**Issue by TheAssassin** [TeamNewPipe/NewPipe#1554]
While working on some merchandise for another project, I thought it was a good idea to create and order some merchandise for NewPipe as well.

Since we have a few great designers in our team, I thought we should ask them to assist with the design.

@theScrabi and I currently plan two articles: we'd like to have some NewPipe stickers with just the logo and the project name, which can be used for laptop lids, etc. Also, we'd like to have some flyers showcasing the NewPipe project, and explaining how people can get involved and contribute.

The formats I have in mind are the following:

 - Flyers: DIN A6, portrait oriented, with a 3mm margin that is cut off during production. The data format is 111x154, end format 105x148. As usual, backgrounds etc. should span over the entire document, i.e., also cover the margin. This is to avoid white space when the flyers are cut to size.
 - Stickers. DIN A8, landscape oriented, with a 3mm margin that is cut off during production. Data format 58x80, end format 52x74. The stickers are produced transparently, any "white" in the final format won't be printed. Therefore, shadows etc. are not a good idea.

The material should be quite neutral, i.e., not focus on the conferences we're going to be at, so that we don't have to dispose any of them after the event is over.

We will order some quite large amounts so that we can give them away on conferences team members will visit. If you are a contributor and have a chance to meet with us, please let us know so we can bring some of those materials for you to take with you.

The merchandise articles will be paid for from the donations we received.

For details regarding the printing company, their data format descriptions etc., please contact us on IRC (#NewPipe on Freenode).

@TobiGr if you have some time in the following weeks, I'd appreciate if you could give this a try. I will create some simple designs soon, too, so that we can order at least some material for the upcoming events (e.g., KDE Akademy, FrOSCon). If you don't have time, just let me know so we can focus on my designs then.

**Comment by theScrabi** [comment]
> For details regarding the printing company, their data format descriptions etc., please contact us on IRC (#NewPipe on Freenode).

Maybe we can also design some cups and T-shirts. I would love the idea of sending these to contributors.

@TheAssassin please be aware of the accountancy. I will talk to you about that privately.

**Comment by theScrabi** [comment]
Merchandise repo here: https://github.com/TeamNewPipe/Merchandise
@TheAssassin @TobiGr you have write access.

**Comment by TheAssassin** [comment]
> Maybe we can also design some cups and T-shirts. I would love the idea of sending these to contributors.

I could imagine some "if you're a contributor, pay the shipping fees and you'll get one sent in your size" or so. Shipping fees around the world can be really high. As long as data is available publicly, though, people can order shirts locally, too. We just need to describe some "corporate design guidelines" to make sure they all look the same.

**Comment by theScrabi** [comment]
Whereas i could tinkck the "TeamNewPipe since 1337" has mor of a Gunnes design :D

**Comment by TobiGr** [comment]
> Maybe we can also design some cups and T-shirts. I would love the idea of sending these to contributors.

I really like the idea. We can use red tiles as background. @TheAssassin Did you already start to work on drafts? I hope to get some basic sketches done until Friday. 

@bagelcake helped with the F-Droid banner. Are you interested in helping us? I know there are other people which helped us with design. Maybe we should ping the too.

**Comment by TheAssassin** [comment]
@TobiGr no, nothing really useful. Basically, a .svg with a red background (color scheme following the website), the logo and the word "NewPipe". I can share it later, if you want.

**Comment by TheAssassin** [comment]
@TobiGr that's what I got so far.

![newpipe-sticker-a8-nonrotated](https://user-images.githubusercontent.com/4503202/43425927-5947adc0-9454-11e8-8d32-b07c537cb03e.png)

(actual size (DIN A8), rendered with 300 dpi)

Source can be found [here](https://transfer.assassinate-you.net/iJLPN/NewPipe-Sticker-A8-nonrotated.svg).

**Comment by theScrabi** [comment]
Do this stickers have to be rectangular? I would like if the sticker would flow a little around the image :)

**Comment by TheAssassin** [comment]
@theScrabi it's all a matter of the price. Free cuts are _really_ expensive, compared to square versions. What's possible is to make the stickers transparent, but that'll mean all white will not be printed, which is not ideal with our logo either.

An alternative to the DIN A* standards size is to use square stickers. The company offers them with side lengths similar to the widths/heights of the DIN A* ones (i.e., 52mm, 74mm, 104mm etc.).

Also, I wanted to mention that we need to hurry up a bit in order for me to be able to take a few of these with me to the upcoming conferences. The deadline's Thursday.

**Comment by theScrabi** [comment]
Yea maybe something like this
![43425927-5947adc0-9454-11e8-8d32-b07c537cb03e](https://user-images.githubusercontent.com/1891273/43455682-fec7ca02-94c0-11e8-962d-cb5f121f517b.png)

**Comment by TheAssassin** [comment]
@theScrabi you can ask there, sure.

At the printing service I use it's 1000 for ~20 EUR and 2500 for ~28-29 EUR square stickers.

The italic font is by the way not the best idea. @TobiGr do we have a CD font? If not, we should decide on one for the logo at least.

**Comment by theScrabi** [comment]
1000 are quite a lot.

**Comment by TheAssassin** [comment]
It's quite easy to give them away on conferences.

**Comment by TheAssassin** [comment]
@theScrabi and I decided to order a lot of 1000 square shaped stickers (52x52mm plus 3mm margin) for now. We can design an improved version later. The design will follow my initial draft.

@TobiGr the deadline is Thursday, although I'd like to order tomorrow. If you want to design something and will be finished by Thursday, we can use your design.

**Comment by TobiGr** [comment]
Sorry for coming back that late. I don't have time to come up with better designs for now.
I think it is a good idea to add a link the our website. I'd suggest somewhere on the right side. It's for people who see this sticker somewhere (toilet, lamp post, laptop or wherever people put these stickers :D) and have no idea what NewPipe is about.

**Comment by theScrabi** [comment]
@TheAssassin nice work. However I think we should design something propper for comming events like CCC, etc.

I would love to have different typrs of stickers. Some highquality once for notebooks, and some informal once with link maybe.

**Comment by justanidea** [comment]
Maybe put a little fdroid logo like that people understand they wont find it on playstore atm...

**Comment by TheAssassin** [comment]
![aufkleber_52x52](https://user-images.githubusercontent.com/4503202/43552669-b596cb90-95eb-11e8-8b15-d91c3fa671cc.png)

Last time for comments, folks. I have to upload this file quite soon. I like it quite much now. I chose white background stickers to make sure the whie parts will actually be white.

Regarding the link to website @TobiGr suggested, I think it's more than easy for people to enter "NewPipe" into their favorite search engine. I tried to add a link, but it was quite disturbing (the URL is quite long after all). It's actually my intention to leave this a bit "cryptic" for non-users. If they're really interested, they'll search for the term. This will hopefully increase our user base.

And also you should remember we're bringing these stickers to events where a lot of people _know_ NewPipe already. I believe this is a really good design. @theScrabi was totally right about using a square shape.

**Comment by TobiGr** [comment]
>  I think it's more than easy for people to enter "NewPipe" into their favorite search engine.

True. Good job!

**Comment by TheAssassin** [comment]
Thanks. By the way, I already thought about getting some shorter like `newpipe.xyz` or `newpipe.io` or perhaps even `np.tld` or so, and make it redirect to our website. @theScrabi what do you think about this?

**Comment by theScrabi** [comment]
newpipe.io sounds good.

**Comment by TheAssassin** [comment]
@theScrabi and it's quite expensive.

**Comment by theScrabi** [comment]
newpipe.com currently has a net worth of about 3000€

**Comment by TheAssassin** [comment]
@theScrabi I was talking about the 69 EUR annual cost for such a `.io` domain.

newpi.pe would be available (Peru's TLD, ~89 EUR). But that's even more expensive.

This "has a net worth of about" stuff is pure bullshit. This practice is called cybersquatting and should IMO be illegal. We probably have established a trademark with NewPipe, and might legally be entitled to claim domain names containing the word "newpipe". I will look into that.

**Comment by TobiGr** [comment]
What about `newpipe.org`? I think it' cheaper and once the project grows further - and I guess it will -  it sounds better and more professional.

**Comment by TheAssassin** [comment]
@TobiGr both `newpipe.org` and `newpipe.net` are still free. _If_ we want either of those, we should get them ASAP, now that we publicly announced interest in them.

**Comment by TobiGr** [comment]
Yes, I agree. @TheAssassin @theScrabi What's your favourite?

**Comment by TheAssassin** [comment]
I like `newpipe.net`.

**Comment by theScrabi** [comment]
newpipe.org 😃

**Comment by TobiGr** [comment]
I'd rather take newpipe.org

**Comment by TheAssassin** [comment]
@theScrabi changed his opinion to `newpipe.net`, so we registered this domain. Check it out: https://newpipe.net

**Comment by TobiGr** [comment]
Awesome!

**Comment by krtkush** [comment]
I was going to recommend newpi.pe but then I see you guys have already registered the .net one.

**Comment by TheAssassin** [comment]
@krtkush it's just a lot more expensive, and we don't want to spend _that_ much on it.

**Comment by krtkush** [comment]
Fair enough!


On 12-Aug-2018 at 7:05 PM, <TheAssassin <notifications@github.com>> wrote:

@krtkush <https://github.com/krtkush> it's just a lot more expensive, and
we don't want to spend *that* much on it.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/TeamNewPipe/NewPipe/issues/1554#issuecomment-412343298>,
or mute the thread
